Echo/Echo.cpp

```diff
@@ -35,59 +36,13 @@
  *
  */
 
-#include <iostream>
-#include <PolySyncNode.hpp>
-#include <PolySyncDataModel.hpp>
+#include <EchoNode.hpp>
```

Comment:
Is EchoNode.hpp a system installed header or within this example project? If it is for this example, then maybe should be `#include "EchoNode.hpp"`